pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -106,5 +107,11 @@ def _trainer_has_checkpoint_callbacks(self):
 
     def attach_model_logging_functions(self, model):
         for callback in self.trainer.callbacks:
+            if at_least_one_overriden(["log", "log_dict"], callback, Callback):
```

Comment:
what would happen if the users has a attribute call `log` (or `log_dict`). We still override, but would the user also be warned in this case?

Reply:
Good question. I don't except user having a property called `log` or `log_dict`, but if they do. It should return an error as we can't assign to property without a setter.

Reply:
Alright, fine by me :]

Reply:
Do you think we should add a check on vars too ?

Reply:
No, I consider this a very edge case.